File: Android/Vine-5.50.1/Decompilation/app/src/main/java/co/vine/android/scribe/model/PostOrRepostDetails.java
package co.vine.android.scribe.model;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
/* loaded from: classes.dex */
public class PostOrRepostDetails {

    @JsonField(name = {"byline"})
    public Byline byline;

    @JsonField(name = {"has_similar_posts"})
    public Boolean hasSimilarPosts;

    @JsonField(name = {"liked"})
    public Boolean liked;

    @JsonField(name = {"longform_id"})
    public String longformId;

    @JsonField(name = {"post_author_id"})
    public Long postAuthorId;

    @JsonField(name = {"post_id"})
    public Long postId;

    @JsonField(name = {"repost_author_id"})
    public Long repostAuthorId;

    @JsonField(name = {"repost_id"})
    public Long repostId;

    @JsonField(name = {"reposted"})
    public Boolean reposted;
}

File: Windows Phone 8.1/2.5.7.0/Decompilation/Vine.WindowsPhone/Vine.Models.Analytics/PostOrRepostDetails.cs
using Newtonsoft.Json;

namespace Vine.Models.Analytics;

public class PostOrRepostDetails
{
	[JsonProperty("post_id")]
	public long PostId { get; set; }

	[JsonProperty("repost_id")]
	public long RepostId { get; set; }

	[JsonProperty("post_author_id")]
	public long PostAuthorId { get; set; }

	[JsonProperty("repost_author_id")]
	public long RepostAuthorId { get; set; }

	[JsonProperty("liked")]
	public bool IsLiked { get; set; }

	[JsonProperty("reposted")]
	public bool IsReposted { get; set; }
}
